### PR TITLE
i18n(zh-cn): Update the translation of from-tauri-1

### PR DESCRIPTION
**Pull request by KikkiZ**
#### Description

- What does this PR change? Updated the translation of from-tauri-1, and, two bugs that existed in the documentation were fixed

### CODE REVIEW

**Comment by KikkiZ** [comment]
Did someone tell me what these failed checks mean and how do I fix them, Thx a lot

**Comment by Embers-of-the-Fire** [comment]
> Did someone tell me what these failed checks mean and how do I fix them, Thx a lot

You can open netlify deploy builds and pull the log to the bottom, where the in-code error is represented.

File: src/content/docs/zh-cn/start/migrate/from-tauri-1.mdx [hunk 1]
```diff
@@ -1169,13 +1171,31 @@ tauri::Builder::default()
 
 `onMenuClicked` 函数已被移除。你可以在 JavaScript 中创建菜单时拦截菜单事件。
 
+### 迁移嵌入的附加文件 (Resources)
+
+在 JavaScript 侧，请确保[迁移到文件系统插件](#迁移到文件系统插件)。
+
+此外，请注意[迁移授权许可](#迁移授权许可)中对 v1 的授权许可列表所作的更改。
+
+在 Rust 侧，请确保[将 Path 迁移到 Tauri Manager](#迁移-Path-到-Tauri-Manager).
```
Comment (Embers-of-the-Fire) [inline src/content/docs/zh-cn/start/migrate/from-tauri-1.mdx]: Broken link.

File: src/content/docs/zh-cn/start/migrate/from-tauri-1.mdx [hunk 2]
```diff
@@ -1169,13 +1171,31 @@ tauri::Builder::default()
...
+
+### 迁移嵌入的外部二进制文档 (Sidecar)
+
+在 Tauri v1 中，外部的二进制文件和其参数在许可列表中被定义。而在 v2 中，使用了新的权限系统。有关详细信息，请阅读[迁移授权许可](#迁移授权许可)。
+
+在 JavaScript 侧，请确保[迁移到 Shell 插件](#迁移到-Shell-插件).
```
Comment (Embers-of-the-Fire) [inline src/content/docs/zh-cn/start/migrate/from-tauri-1.mdx]: Broken link.

**Review by Embers-of-the-Fire — APPROVED** [review]
lgtm, but you still need to wait for an admin's review to approve.

**Review by FabianLars — APPROVED** [review]

**Pull request merged**